src/components/Locationitem/LocationItem.jsx

```diff
@@ -0,0 +1,19 @@
+import PropTypes from "prop-types";
+import RightArrow from "../../assets/icons/right-arrow.png";
+import "./LocationItem.css";
+
+const LocationItem = ({ handleLocation, text }) => {
```

Comment:
very nitpick, but as I said, prefer to put values first and methods after:

```suggestion
const LocationItem = ({ text, handleLocation }) => {
```